modules/check_in/spec/request/v2/appointments_request_spec.rb

```diff
@@ -235,10 +245,72 @@
         end
 
         it 'returns error' do
-          VCR.use_cassette 'check_in/facilities/get_facilities_500' do
-            VCR.use_cassette 'check_in/appointments/get_appointments_200' do
-              VCR.use_cassette 'check_in/map/security_token_service_200' do
-                get "/check_in/v2/sessions/#{id}/appointments", params: { start: start_date, end: end_date }
+          VCR.use_cassette 'check_in/clinics/get_clinics_500' do
+            VCR.use_cassette 'check_in/facilities/get_facilities_500' do
+              VCR.use_cassette 'check_in/appointments/get_appointments_200' do
+                VCR.use_cassette 'check_in/map/security_token_service_200' do
+                  get "/check_in/v2/sessions/#{id}/appointments", params: { start: start_date, end: end_date }
+                end
+              end
+            end
+          end
+
+          expect(response).to have_http_status(:bad_request)
+          expect(response.body).to eq(error_response)
+        end
+      end
+
+      context 'when facility service returns 500 and clinic service succeeds' do
+        let(:error_response) do
+          {
+            errors: [
+              {
+                title: 'Operation failed',
+                detail: 'Operation failed',
+                code: 'VA900',
+                status: '400'
+              }
+            ]
+          }.to_json
+        end
+
+        it 'returns error' do
+          VCR.use_cassette 'check_in/clinics/get_clinics_200' do
+            VCR.use_cassette 'check_in/facilities/get_facilities_500' do
+              VCR.use_cassette 'check_in/appointments/get_appointments_200' do
+                VCR.use_cassette 'check_in/map/security_token_service_200' do
+                  get "/check_in/v2/sessions/#{id}/appointments", params: { start: start_date, end: end_date }
+                end
+              end
+            end
+          end
+
+          expect(response).to have_http_status(:bad_request)
+          expect(response.body).to eq(error_response)
+        end
+      end
+
+      context 'when facility service succeeds 500 but clinic service returns 500' do
```

Comment:
this test needs to be reworded

Reply:
Sorry, I fixed the typo in [3b0ce1a](https://github.com/department-of-veterans-affairs/vets-api/pull/17642/commits/3b0ce1aae8965571b1f8677904383f8199a65367)